ietf/templates/nomcom/year_index.html

```diff
@@ -1,9 +1,13 @@
 {% extends "nomcom/nomcom_public_base.html" %}
-{# Copyright The IETF Trust 2015, All Rights Reserved #}
+{# Copyright The IETF Trust 2015-2024, All Rights Reserved #}
 {% load origin %}
 {% block subtitle %}- Home{% endblock %}
 {% block nomcom_content %}
     {% origin %}
     <!-- [html-validate-disable-block heading-level -- FIXME: old HTML in the database] -->
     {{ template|safe }}
+    {% if nomcom.ann_url %}
```

Comment:
This should probably come before rendering the template, and it probably needs more dressing.
"Announcements from this nomcom are available <here>"
or something like it.
Maybe we need a block in the base template for it instead of intruding on the nomcom_content block.

Reply:
No preference.